modules/nf-core/emmtyper/main.nf

```diff
@@ -33,10 +34,31 @@ process EMMTYPER {
         gzip -c -d $fasta > $fasta_name
     fi
 
-    emmtyper \\
-        $options.args \\
-        $fasta_name \\
-        > ${prefix}.tsv
+    echo $db
+
+    # Conditionally add the database if it is provided by user
+    if [ "$db" == "" ]; then
+        emmtyper \\
+            $options.args \\
+            $fasta_name \\
+            > ${prefix}.tsv
+    else
+
+        # Make the blast database
+        makeblastdb -in $db -dbtype nucl
+
+        emmtyper \\
+            --blast_db $db \\
+            $options.args \\
+            $fasta_name \\
+            > ${prefix}.tsv
+    fi
+
```

Comment:
Can we add a "clean up" section here where the temporary blast files are deleted?